### DISCUSSION TITLE
support for handlebars in v2

### DISCUSSION THREAD

**Discussion by tglman** [parcel-bundler/parcel#6125] [Q&A]
Hi,

Is there any way to have handlebars support in v2 ? I guess the v1 plugins do not work anymore with v2.

**Comment by mischnic** [comment]
> Is there any way to have handlebars support in v2

No

> I guess the v1 plugins do not work anymore with v2.

Correct

**Reply by tglman** [reply]
Hi,

Ok, is the handlebars support completely escluded ? by that i mean not even accept pull request for the support

Would be possible to write an external plugin ?

I guess is possible to just add a transformer in the configuration:
```
...
    "*.hbs": ["custom-hbs-transformer"],
...
```
and have a external dependency that handle the case, if that will work i may progress with my own integration.

Sorry for silly questions bit a newbie of parcel & js.

**Reply by mischnic** [reply]
> Ok, is the handlebars support completely escluded ? by that i mean not even accept pull request for the support

It's just not implemented (yet). I just didn't want to write "Not yet" as that would sound like "we're already working on it".

We are not against PRs adding new file types.

> Would be possible to write an external plugin ?

That would also be possible, just like you described (though the package name isn't quite right: https://v2.parceljs.org/plugin-system/authoring-plugins/)
